lib/public/views/Tags/Details/TagDetailsModel.js

```diff
@@ -226,19 +236,45 @@ class TagDetailsTabbedPanelModel extends TabbedPanelModel {
         }
     }
 
+    /**
+     * Returns the pagination model
+     *
+     * @return {PaginationModel} the pagination model
+     */
+    get pagination() {
+        return this._pagination;
+    }
+
     /**
      * Fetches the logs related to the current tag
      *
+     * @param {boolean} clear if true, any previous data will be discarded, even in infinite mode
      * @returns {Promise<void>} resolves once the data has been retrieved
      * @private
      */
-    async _fetchLogsPanelData() {
+    async _fetchLogsPanelData(clear = false) {
+        /**
+         * @type {Run[]}
+         */
+        const concatenateWith = !clear && this._pagination.currentPage !== 1 && this._pagination.isInfiniteScrollEnabled
+            ? this._currentPageRuns.payload || []
+            : [];
+
         this.currentPanelData = RemoteData.loading();
```

Comment:
The currentPnaleData contains the content of the table. 
I believe you want to change the state and display a loading icon, only if the user uses pagination and not infinite mode.